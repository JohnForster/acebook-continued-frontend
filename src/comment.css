.comment {
  background-color: #EAEBF0;
  font-size: 13px;
  padding: 8px;
}

.comment_footer {
  color: #A0A1A5;
}

.comment_header_username {
  display: inline;
  padding: 0 10px 0 0;
}

.comment_header_body {
  display: inline;
}