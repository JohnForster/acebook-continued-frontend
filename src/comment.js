import React from 'react'

export class Comment extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      body: props.body,
      time: props.time,
      user: props.user_id
    }
  }

  render() {

    let body = this.state.body;
    let time = this.state.time;
    let user = this.state.user;
    return(
      <div> {body}
            {time}
            {user}
     </div>
    )
  }
}
