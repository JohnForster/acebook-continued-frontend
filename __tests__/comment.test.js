import React from 'react';
import expect from 'expect';
import 'isomorphic-fetch';
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme';
import {Comment} from '../src/comment.js';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
enzyme.configure({ adapter: new Adapter() });

describe('Comment', () => {
  it('Post displays comment', () => {
    const component = shallow(<Comment message='Nice post' />);
    expect(component).toMatchSnapshot();
  });
});

// const component = renderer.create(<Post message='Hello' time='Today' />);
