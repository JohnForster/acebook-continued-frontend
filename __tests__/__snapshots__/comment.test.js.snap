// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Post displays comment 1`] = `
<div
  className="post"
>
  Hello
   
  <br />
   (posted at 
  Today
  )
</div>
`;
